src/collective/taxonomy/browser.py

```diff
@@ -1,6 +1,13 @@
+import Missing
 from collective.taxonomy.interfaces import ITaxonomy
+from plone import api
+from plone.app.contenttypes.browser.collection import CollectionView
+from plone.app.vocabularies.metadatafields import get_field_label
```

Comment:
This is not available in Plone < 5.2 (plone.app.vocabularies < 4.2) should we remove support for Plone < 5.2?

Reply:
no we shouldn't 😉